src/skirmishbunker/DetailedRoof.py

```diff
@@ -0,0 +1,186 @@
+import cadquery as cq
+from .FlatRoof import FlatRoof
+from cadqueryhelper import series
+from cqterrain import roof
+from math import floor as math_floor
+
+class DetailedRoof(FlatRoof):
+    def __init__(self):
+        super().__init__()
+
+        self.bunker_int_length = None
+        self.bunker_int_width = None
+        self.angle = 0
+
+        self.wall_details_inset = 3
+        self.wall_details_depth = 5
+        self.wall_arch_fillet = 2
+
+        self.hatch_cut_inset = 2
+        self.hatch_cut_chamfer = 2
+
+        self.roof_overflow = 0
+
+        self.outline = None
+        self.cut_walls = None
+        self.wall_details = None
+
+    def calc_tile_z_translate(self):
```

Comment:
Commented on these calc methods in another comment. My gut says these should be protected or private.

Reply:
Like I said previously, I would probably say protected.